nss-crystal-cobras/BangazonSite
# Product sell#3 jd

# Description

Closes Ticket #3 (https://github.com/nss-crystal-cobras/BangazonSite/issues/3)

Given the user is not authenticated
When any view renders
Then the Sell a Product affordance will not be visible

Given the user is authenticated
When the user clicks on the Sell a product button in the menu bar
Then the user will be presented with a product form

Given the user has filled in all fields of the product form with valid information
When the user clicks the Sell button
Then the user will be redirected to the product detail view for that product

Given the user has filled in all fields of the product form with any invalid information
When the user clicks the Sell button
Then the user will be immediately notified with error information for each invalid value

## Type of change


# Testing Instructions

git fetch --all
git checkout ProductSell#3JD  (TRUST ME)

To test:
-- Log out and you will NOT see nav bar link to "Sell Something"
-- Log in and you will see nav bar link
-- Click link fill out forms. 
-- Forms are validated if you miss any info.
-- Click create it will take you to the item's details.
-- To list shows the item at the bottom of the list.
-- Are you not entertained?

# Checklist:
